test/framework/src/main/java/org/opensearch/test/OpenSearchSingleNodeTestCase.java

```diff
@@ -242,6 +244,7 @@ private Node newNode() {
             .put(HierarchyCircuitBreakerService.USE_REAL_MEMORY_USAGE_SETTING.getKey(), false)
             .putList(DISCOVERY_SEED_HOSTS_SETTING.getKey()) // empty list disables a port scan for other nodes
             .putList(INITIAL_CLUSTER_MANAGER_NODES_SETTING.getKey(), nodeName)
+            .put(TelemetrySettings.TRACER_ENABLED_SETTING.getKey(), true)
```

Comment:
I don't think this should be the default, at least at the moment

Reply:
The idea is to do strict checking of the spans like all the spans should be closed to avoid any memory leak possibilities. That's why added it as default. Anyways it can be overridden through node settings if needed.

Reply:
Fair enough, catching any issues during test runs is certainly better